### PR TITLE
NEW triggers add commercial and del commercial

### PR DESCRIPTION
**Pull request by atm-quentin**
# New triggers add commercial and del commercial

### CODE REVIEW

File: htdocs/societe/class/societe.class.php [hunk 1]
```diff
@@ -1924,6 +1929,12 @@ function add_commercial(User $user, $commid)
 			{
 				dol_syslog(get_class($this)."::add_commercial Erreur");
 			}
+			else {
+				$this->commercial_modified = $commid;
```
Comment (eldy) [inline htdocs/societe/class/societe.class.php]: Parameters to a triggers must be passed with
$this->context=array('commercial_modified'=>$commid);

File: htdocs/societe/class/societe.class.php [hunk 2]
```diff
@@ -1924,6 +1929,12 @@ function add_commercial(User $user, $commid)
...
+				
+				$result=$this->call_trigger('COMPANY_ADD_COMMERCIAL',$user);
```
Comment (eldy) [inline htdocs/societe/class/societe.class.php]: Can you name your trigger 'COMPANY_LINK_SALE_REPRESENTATIVE' to match naming rules used by other objects

File: htdocs/societe/class/societe.class.php [hunk 3]
```diff
@@ -1936,6 +1947,12 @@ function add_commercial(User $user, $commid)
 	 */
 	function del_commercial(User $user, $commid)
 	{
+		$error=0;
+		$this->commercial_modified = $commid;
+				
+		$result=$this->call_trigger('COMPANY_DEL_COMMERCIAL',$user);
```
Comment (eldy) [inline htdocs/societe/class/societe.class.php]: Can you name your trigger 'COMPANY_UNLINK_SALE_REPRESENTATIVE' to match naming rules used by other objects (this is not a CRUD deletion)

File: htdocs/societe/class/societe.class.php [hunk 4]
```diff
@@ -1936,6 +1947,12 @@ function add_commercial(User $user, $commid)
 	 */
 	function del_commercial(User $user, $commid)
 	{
+		$error=0;
+		$this->commercial_modified = $commid;
```
Comment (eldy) [inline htdocs/societe/class/societe.class.php]: Parameters to a triggers must be passed with
$this->context=array('commercial_modified'=>$commid);

**Comment by atm-quentin** [comment]
up

**Pull request merged**